licenses/LICENSE-graal.txt

```diff
@@ -0,0 +1,9 @@
+          Product License  - GraalVM Community Edition
+
+This is a release of GraalVM Community Edition. GraalVM Community Edition
+consists of multiple modules. The software as a whole, as well as the JVMCI
+and VisualVM modules, are released under version 2 of the GNU General Public
+License with the “Classpath” Exception.
+The text of the foregoing licenses is reproduced below.
+
+Copyright (c) 2015, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is not the full text of the Graal license. Please copy all (https://github.com/oracle/graal/blob/master/LICENSE) here.

Reply:
Done